### DISCUSSION TITLE
Option to filter for guys based on whether they would help you/pay for you to get an abortion

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#104] [Ideas & Features]
Given the current political climate, a lot of women will want the assurance that the guy they date/hookup with will help them if they get in this situation, and we should facilitate that.

**Comment by ajbott** [comment]
I believe a dynamic filter would help people determine what matters most to people. If people are concerned with having a partner that would be willing to help pay/support you through an abortion or even if they are on the pro-life/pro-choice side, people should have the option to be able to filter to find them.
